opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/batcher.py

```diff
@@ -86,24 +56,16 @@ def finished_collection(self):
         if not self.stateful:
             self._batch_map = {}
 
-    @abc.abstractmethod
     def process(self, record) -> None:
-        """Stores record information to be ready for exporting.
-
-        Depending on type of batcher, performs pre-export logic, such as
-        filtering records based off of keys.
-        """
-
-
-class UngroupedBatcher(Batcher):
-    """Accepts all records and passes them for exporting"""
-
-    def process(self, record):
+        """Stores record information to be ready for exporting."""
         # Checkpoints the current aggregator value to be collected for export
-        record.aggregator.take_checkpoint()
-        batch_key = (record.instrument, record.labels)
-        batch_value = self._batch_map.get(batch_key)
         aggregator = record.aggregator
+        aggregator.take_checkpoint()
+
+        # The uniqueness of a batch record is defined by a specific metric
+        # using an aggregator type with a specific set of labels.
+        key = (record.instrument, aggregator.__class__, record.labels)
```

Comment:
Doesn't this mean you can't aggregate the same metric with e.g. two different histograms with different bucket boundaries?